tests/active_features/bootstrap/FeatureContext.php

```diff
@@ -150,7 +150,7 @@ public function iAddHostnameToTheEnvironmentOf($hostname, $env, $site)
      */
     public function iAddToTheTeamOn($email, $site)
     {
-        $this->iRun("terminus site:team:add --site=$site --member=$email");
+        $this->iRun("terminus site:team:add $site --member=$email");
```

Comment:
Cool, I've fixed that and the other places where it was wrong in the FeatureContext file.